### PR TITLE
Add wallet extension links to app pages

### PR DESCRIPTION
**Pull request by candida-d**
Links to the fairground extensions for the Vega Wallet

### CODE REVIEW

File: docs/tools/vega-wallet/index.md [hunk 1]
```diff
@@ -8,7 +8,19 @@ import Topic from '/docs/topics/_topic-wallet.mdx'
 
 A Vega Wallet is essential for interacting with Vega, whether it's for staking or trading. The Vega Wallet apps allow you to manage wallets and key pairs, deposit and withdraw assets, stake and sign transactions.
 
-You can interact with a Vega wallet and its keys through two different apps, or integrate using the API. 
+You can interact with a Vega wallet and its keys through a browser extension, two different apps, or integrate using the API.
+
+## Vega Wallet browser extension
+The Vega Wallet browser extension is an early stage implementation of the Vega Wallet, available  for Chrome and Firefox. Right now the extensions are only available in stores for use with the Fairground network.
```
Comment (edd) [inline docs/tools/vega-wallet/index.md:14]:
```suggestion
The Vega Wallet browser extension is an early stage implementation of the Vega Wallet, available for Chrome and Firefox. Right now the extensions are only available in stores for use with the Fairground network.
```

the nittiest pick

File: docs/tools/vega-wallet/index.md [hunk 2]
```diff
@@ -8,7 +8,19 @@ import Topic from '/docs/topics/_topic-wallet.mdx'
...
+
+**[Chrome extension ↗](https://chrome.google.com/webstore/detail/vega-wallet-fairground/nmmjkiafpmphlikhefgjbblebfgclikn)**
+
+**[Firefox extension ↗](https://addons.mozilla.org/en-GB/firefox/addon/vega-wallet/)**
+
+The browser extension lets you connect to a Vega Wallet right in your browser, rather than having to install a separate app. It allows you to:
+* Create a new wallet and public keys
+* Import an existing wallet and its keys
+* Connect to Vega dApps, such as Console and the governance dApp
```
Comment (edd) [inline docs/tools/vega-wallet/index.md:23]: **praise (non-blocking)**: Obviously it does more, but I think you've selected the right level to say 'most things you want'

**Review by edd — APPROVED** [review]
One non-blocking suggestion (removing a double space (`  `)

**Review by daunatv — COMMENTED** [review]
The browser extension lets you connect to a Vega Wallet right in your browser, rather than having to install a separate app. It allows you to:
* Create a new wallet and public keys
* Import an existing wallet and its keys
* Connect to Vega dApps, such as Console and the governance dApp

I would probably still add approve transactions or something similar since that would cover transfers and trades.
@candida-d @candida-d

**Pull request merged**